File: LeetCodeProblemsGo/LeetCode3190FindMinimumOperationsToMakeAllElementsDivisibleByThree/LeetCode3190FindMinimumOperationsToMakeAllElementsDivisibleByThree.go
package LeetCode3190FindMinimumOperationsToMakeAllElementsDivisibleByThree

func minimumOperations(nums []int) int {
	operationsCount := 0
	for _, num := range nums {
		if num%3 != 0 {
			operationsCount++
		}
	}
	return operationsCount
}

File: LeetCodeProblemsCMake/Problems/LeetCode3190FindMinimumOperationsToMakeAllElementsDivisibleByThree.cpp
//
// Created by saeed on 11/22/25.
//

#include "LeetCode3190FindMinimumOperationsToMakeAllElementsDivisibleByThree.h"
using namespace std;
int LeetCode3190FindMinimumOperationsToMakeAllElementsDivisibleByThree::minimumOperations(vector<int> &nums) {
    int operationsCount{};
    for (auto num : nums) {
        if (num%3 != 0) {
            operationsCount++;
        }
    }
    return operationsCount;
}
